packages/dso-toolkit/src/components/logo/logo.mixins.scss

```diff
@@ -0,0 +1,109 @@
+@use "../../variables/units";
+@use "../../variables/colors";
+@use "../../variables/typography";
+@use "logo.variables" as logo-variables;
+
+@mixin root() {
```

Comment:
De mixins maken we alleen als er meerdere implementaties zijn en het handig is om styling te delen. Dit component wordt alleen geimplementeerd als Web Component.

Variables mogen wel in deze package blijven staan, maar dat laat ik aan jou :+1: